apps/www/_blog/2023-04-10-o11y-day.mdx

```diff
@@ -0,0 +1,207 @@
+
+To expose log data to customers, we leverage Logflare Endpoints. This provides an HTTP integration into Supabase Studio, powering the log query UIs and most timeseries charts. These charts live across the studio, such as the project home page and the new API reports.
+
+{/* <!--  add chart screenshot --> */}
+
+## Self-hosting Logflare
+
+Logflare was available under a BSL license prior to joining Supabase. We’ve since changed the license to [Apache 2.0](https://github.com/Logflare/logflare/blob/staging/LICENSE.md), aligning it with our open source philosophy.
+
+In the past few months we’ve made Logflare more developer-friendly for local development and self-hosting. While you’re building a project, you can view and query your logs from any Supabase service, just as you would in our cloud Platform.
+
+![Realtime local logs](/images/blog/lw7-o11y/realtime-logs-screenshot.png)
+
+> 📢 Check out the [new self-hosting docs](http://localhost:3001/docs/reference/self-hosting-analytics/introduction) to get Logflare up and running as your analytics server.
+>
+> It currently supports a BigQuery backend, and we are actively working on supporting more.
+
+## The Ingestion Pipeline
+
+Logflare receives Supabase log events via multiple methods. Services like Postgres use Vector to clean and forward log events to our the Logflare ingest API. Other services such as Realtime and Storage utilize our [integrations](https://github.com/Logflare/logflare#integrations) to send the log events directly. These then get processed and streamed into BigQuery.
+
+## The Querying Pipeline
+
+The hard part comes after ingesting the logs: searching, aggregating ,and analyzing them at scale. Crunching many terabytes of data on each query is expensive, and exposing the ingested data to Supabase customers in a naive manner would cause our costs to skyrocket.
+
+To solve these issues, we built and refined Logflare Endpoints, the query engine that powers many of Supabase’s features, such as the logs views, Logs Explorer, and usage charts.
+
+With Endpoints, you can create HTTP API endpoints from a SQL query, including parameterized queries. Endpoints is like a PostgREST view but with some benefits:
+
+- **Query parameters**
+  - You can provide string parameters to the SQL query via the HTTP endpoint.
+- **Read-through caching**
+  - Results from the query are cached in memory for fast response times.
+  - A read-through cache provides results if cached results do not exist.
+- **Active cache warming**
+  - Query results are proactively warmed at a configurable interval for a combination of fast response times and as-realtime-as-needed data.
+- **Query sandboxing**
+  - If an Endpoint query contains a CTE and the sandbox option is selected, the Endpoint will inject the query string of the `sql` query parameter into the Endpoint SQL replacing the default query (the part of the SQL query after the CTE).
```

Comment:
this one is still very confusing @Ziinc - I don't understand exactly what it means